src/dist/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 add_subdirectory(failure_detector)
 add_subdirectory(replication)
 add_subdirectory(cluster_scheduler)
-add_subdirectory(deployment_service)
+#add_subdirectory(deployment_service)
```

Comment:
because in deploy_svc_service_impl::stop(), the gpid() is changed to
get_gpid(), but the method is not exist, and compile error. So I leave it
to you to fix.

On Thu, Jun 16, 2016 at 11:44 AM, Zhenyu Guo <notifications@github.com>
wrote:

> In src/dist/CMakeLists.txt
> <https://github.com/imzhenyu/rDSN/pull/403#discussion_r67283444>:
>
> > @@ -1,7 +1,7 @@
> >  add_subdirectory(failure_detector)
> >  add_subdirectory(replication)
> >  add_subdirectory(cluster_scheduler)
> > -add_subdirectory(deployment_service)
> > +#add_subdirectory(deployment_service)
>
> why?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/imzhenyu/rDSN/pull/403/files/89a41d5e513a99014291d1ad76187ae04459b1ab#r67283444>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAgQPaVd3GjPLFkrgwh0Wxo4xPF2QNeBks5qMMacgaJpZM4I2J8D>
> .
>